compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/TupleMemberSymbol.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.ballerina.compiler.api.symbols;
+
+/**
+ * Represents a tuple member symbol.
+ *
+ * @since 2201.4.0
+ */
+public interface TupleMemberSymbol extends Symbol, Annotatable {
+
+    /**
+     * Get the type of the member.
+     *
+     * @return {@link TypeSymbol}
+     */
+    TypeSymbol getType();
```

Comment:
Wouldn't it be better to use `typeDescriptor()` here?
Also, why can't we restrict the return type to `TupleTypeSymbol`?
Since that's how it has been named in other similar places within the Semantic API.
```suggestion
    TupleTypeSymbol typeDescriptor();
```

Reply:
Renaming the `getType()` to `typeDescriptor()` is okay. But, why we should cast the TypeSymbol to TupleTypeSymbol ? we don't always get a tuple-type

Reply:
Okay. Got it. Shall we proceed with the method name renaming?

Reply:
yeah +1